roundtripper/roundtripper.go

```diff
@@ -19,18 +19,19 @@ import (
 	"log"
 	"net/http"
 	"os"
-	"strconv"
 	"strings"
 
-	"github.com/bradleyfalzon/ghinstallation"
 	"go.uber.org/zap"
 
 	"github.com/ossf/scorecard/clients/githubrepo"
 )
 
+// nolinter
+// GithubAuthToken is for making requests to GiHub's API.
```

Comment:
`GithubAuthToken` should be plural in comments to match the var name. Also remove `// nolinter`, which you shouldn't need after updating to plural.

Reply:
done.